### DISCUSSION TITLE
Footnotes

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#270] [Get Help] [answered]
Hi all! Newbie here just feeling my way through Quire!

I'm trying to figure out the footnotes. I was following the footnote instructions on this page: [https://quire.getty.edu/documentation/fundamentals/#markdown-basics](https://quire.getty.edu/documentation/fundamentals/#markdown-basics)

I created my footnote in the text and added a Notes section at the bottom in my markdown file:
![Screen Shot 2022-03-21 at 5 18 11 PM](https://user-images.githubusercontent.com/74433007/159372938-dd755644-7564-4b73-8083-b9bd56c43063.png)
Notice that the word "testing" is grayed out. I don't know why. When I save, I get a drop cap for the word "testing":
![Screen Shot 2022-03-21 at 5 18 45 PM](https://user-images.githubusercontent.com/74433007/159373054-3f765e62-9631-4dcf-accb-e46005579819.png)

I don't want drop caps in my footnotes, but I'm not sure how to fix this. The footnote I'm playing with is on a Splash page which has drop caps in the first graf. But I don't know why that would apply to the notes.

Any advice would be much appreciated!

**Comment by geealbers** [comment, marked as answer]
hi @mmpadilla15. You've discovered a bug in the Quire stylesheets. 🐞 Apparently no one else has tried to add footnotes to a splash page! I tested a fix that seems to work: On lines 662 and 681 of themes/default/source/css/components/quire-page.scss change `p` to `.content > p`. Save the change and you may need to stop and restart the preview if it doesn't load.

Let us know if it works and we'll push this fix to the Quire repo as well. Thanks!

**Reply by mmpadilla15** [reply]
@geealbers Yes - that worked! Thank you! One thing to note, in my quire-page.scss page, the lines to change were 662 and **680**.

![Screen Shot 2022-03-24 at 11 56 39 AM](https://user-images.githubusercontent.com/74433007/159969705-44daff5a-8f24-48ab-a9cf-ae54b71abd5f.png)
